phcovid/phcovid.py

```diff
@@ -52,11 +68,30 @@ def parse_numeric(s):
     return case_list
 
 
-def get_cases(
-    rename_dict=RENAME_DICT, val_alias=VAL_ALIAS, none_alias=NONE_ALIAS
-):
+def attach_supplement_data(dataframe):
+    # change targets to extend the dataset
+    targets = [
+        "case no.",  # for row searching
+        "status",
+        "symptoms",
+    ]
+    missing = extract_dsph_gsheet_data(target_columns=targets)
+
+    for target in targets[1:]:
```

Comment:
Can we not change the dataframe inplace? Once `targets` is set as an argument. We can explicitly edit those columns in `get_cases`

Reply:
Done and modified for the inplace df modification.